src/swarm/neo/client/mixins/RequestCore.d

```diff
@@ -375,6 +375,57 @@ public template RequestCore ( RequestType request_type_, ubyte request_code,
         assert(false);
     }
 
+    /***************************************************************************
+
+        Private helper function to handle notification of the global supported
+        codes, including the RequestSupported code.
+
+        Params:
+            status = code received from node
+            context = deserialized request context
+            remote_address = address of node which returned this status code
+
+        Returns:
+            false if `status` was a global error code and the user notified;
+            true if `status` is a RequestSupported code indicating that the
+            request is supported and that the request will start.
+
+    ***************************************************************************/
+
+    private static bool handleSupportedCodes ( StatusCode status,
```

Comment:
As we no longer intend to allow the set of codes to be "extended" per request type, this function should just accept `SupportedStatus` directly.